app/src/ui/add-repository/create-repository.tsx

```diff
@@ -446,6 +446,9 @@ export class CreateRepository extends React.Component<
         ariaLiveMessage={`Will be created as ${sanitizedName}. Spaces and invalid characters have been replaced by hyphens.`}
       >
         <p>Will be created as {sanitizedName}</p>
+        <span className="sr-only">
```

Comment:
Note: This change was after my screenshares as I realized focusing on the input did not announce the same as the announcement during typing, but behavior is the same just messaging change.